src/Command/DrupalUpdate/CheckInfo.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+
+namespace Acquia\Cli\Command\DrupalUpdate;
+
+class CheckInfo
```

Comment:
Seems like this needs a better name. Check what info? Info for application packages?